components/automate-ui/src/app/page-components/projects-filter-dropdown/projects-filter-dropdown.component.html

```diff
@@ -11,7 +12,7 @@
   </span>
   <chef-icon *ngIf="dropdownCaretVisible" aria-hidden>keyboard_arrow_down</chef-icon>
 </button>
-<chef-dropdown class="dropdown" *ngIf="dropdownActive" (keydown.esc)="handleEscape()">
+<chef-dropdown class="dropdown" data-cy="projects-filter-dropdown" *ngIf="dropdownActive" (keydown.esc)="handleEscape()">
   <chef-click-outside class="dropdown-body" omit="dropdown-label" (clickOutside)="handleEscape()">
```

Comment:
Consider moving these cypress annotations to the separate cypress PR

Reply:
i agree it's awkward they're here. however we want these to get into acceptance before we merge those cypress tests, since they'll fail until these changes are in place.

Reply:
Sounds good. 👍